### PR TITLE
Bump the rabbitmq helm chart to 6.19.2

### PR DESCRIPTION
**Pull request by tczekajlo**
Bump up the rabbitmq chart to 6.19.2

### CODE REVIEW

File: charts/rasa-x/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 
-version: "1.4.1"
+version: "1.4.2"
```
Comment (wochinge) [inline charts/rasa-x/Chart.yaml:4]: I think we should bump to `1.5` as we are changing dependencies, wdyt?

File: charts/rasa-x/Chart.lock [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 - name: redis
   repository: https://charts.bitnami.com/bitnami
-  version: 10.5.12
+  version: 10.5.14
 - name: rabbitmq
   repository: https://charts.bitnami.com/bitnami
-  version: 6.18.9
-digest: sha256:29407b1674ec137a7123b46bb549039321efb842e07bf6025ac3a2324e089477
-generated: "2020-03-19T08:55:27.193432+01:00"
+  version: 6.19.2
```
Comment (wochinge) [inline charts/rasa-x/Chart.lock:10]: did you check what they changed from 6.18 to 6.19? Hopefully didn't rename any `values.yml` keys, did they?

Comment (tczekajlo) [reply charts/rasa-x/Chart.lock:10]: I did, they added only two fields, `extraVolumeMounts` and `extraVolumes`. Those two fields us a reason why we bump a rabbitmq chart version.

File: charts/rasa-x/Chart.yaml (hunk 1 above)
Comment (tczekajlo) [reply charts/rasa-x/Chart.yaml:4]: Good point

**Comment by tczekajlo** [comment]
> 🚀
> 
> * could you please update the links to the subcharts in the readme?

Hmm, I don't see any links to the subcharts in the readme that I can update 😄  or I'm blind :D

**Comment by wochinge** [comment]
> Hmm, I don't see any links to the sub-charts in the readme that I can update 😄 or I miss something

Sorry, I mean in the `values.yml` 🙈

**Pull request merged**